app/Policies/CommentPolicy.php

```diff
@@ -26,6 +28,37 @@ public function view(?User $user, Comment $comment): bool
         return true;
     }
 
+    public function create(User $user, ?Model $commentable): bool
+    {
+        if ($user->isMuted()) {
+            // Even when muted, developers may still comment on tickets for their own achievements.
```

Comment:
Should this be limited to unresolved tickets?
`TicketState::isOpen($commentable->ReportState)`

I've had users thank me after resolving a ticket, so I assume we'd want to allow two-way communication.

Reply:
Similarly, should a muted user be allowed to comment on an achievement they authored, or the related game?

Reply:
Frankly, I think we're already being pretty generous just with this change consider a target user will actively be under moderation.

I very much support the idea of locking this down further to only tickets that are still open and have made that change in latest. Personally, I don't think we should extend this to authored achievements or associated games unless the team feels it's an absolute necessity.